systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaST.java

```diff
@@ -542,7 +542,7 @@ void testLabelsExistenceAndManipulation(ExtensionContext extensionContext) {
             .endSpec()
             .build());
 
-        resourceManager.createResource(extensionContext, KafkaTopicTemplates.topic(testStorage.getClusterName(), testStorage.getTopicName()).build());
+        resourceManager.createResource(extensionContext, KafkaTopicTemplates.topic(testStorage).build());
```

Comment:
I am not sure if I like this approach. It makes the code a little bit mysterious as you pass there only testStorage but it is my opinion.

Reply:
We already do this with kafkaUsers, but i agree it's not instantly telling.